charts/internal/control-plane/templates/duros-controller.yaml

```diff
@@ -87,6 +88,8 @@ spec:
       labels:
         app: duros-controller
     spec:
+      # required to be able to read the duros cr from the seed
+      automountServiceAccountToken: true
```

Comment:
The gardener PRs do not have this field set on similar controllers and instead have `automountServiceAccountToken: false` on the related service accounts. Not sure if this works only in 1.45 but in the end we should also do it this way because the service account tokens are also rotated by the GRM.